src/model/annotation-list.v1.yaml

```diff
@@ -0,0 +1,89 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Annotation list
+type: object
+properties:
+    total:
+        title: Total number of Annotations
+        type: integer
+        minimum: 0
+    items:
+        title: Annotations
+        type: array
+        items:
+            type: object
+            properties:
+                id:
+                    description: Annotation ID
+                    type: string
+                access:
+                    $ref: ../misc/access-level.v1.yaml
+                hasParent:
+                    description: If the annotation has a parent annotation
+                    type: boolean
+                    default: false
+                highlight:
+                    description: Highlighted text
+                    type: string
+                published:
+                    description: Publication date (UTC)
+                    $ref: ../misc/date-time.v1.yaml
+                updated:
+                    title: Updated date (UTC)
+                    $ref: ../misc/date-time.v1.yaml
+                content:
+                    title: Content
+                    type: array
+                    items:
+                        oneOf:
+                          - $ref: ../blocks/list.v2.yaml
+                          - $ref: ../blocks/mathml.v1.yaml
+                          - $ref: ../blocks/paragraph.v1.yaml
+                          - $ref: ../blocks/quote.v1.yaml
+                          - $ref: ../blocks/youtube.v1.yaml
+                item:
+                    oneOf:
```

Comment:
in order to introduce these snippets we are going to have to store them as part of the annotations service or perform an API request for each annotation, so we might need to rethink this? @giorgiosironi @thewilkybarkid

Reply:
How many annotations are we talking about in a single response?

Recommendations uses the approach of making several internal API requests, without cache, to compose a result. However, they are not that many, ~5 requests per API response tops.
Search is doing the same but with caching (I think?) or is it actually persisting in ElasticSearch the snippets? Will check.

Just to understand the whole context, where does the requirement of serving the snippets came from?

Reply:
Each annotation will have a snippet. So, if you request 20 annotations in a list then there will be up to 20 requests.

Reply:
Search is not caching any API call explicitly, the snippets are stored in ElasticSearch and overwritten on bus events. Which is what `annotations` would have to do, with a different persistence storage probably.